src/bsoncxx/json.hpp

```diff
@@ -36,28 +36,6 @@ BSONCXX_INLINE_NAMESPACE_BEGIN
 BSONCXX_API std::string BSONCXX_CALL to_json(document::view view);
 
 ///
-/// Converts an element (key-value pair) to a JSON key-value pair.
-/// This is not actually a valid JSON document, but it can be useful for debugging.
-///
-/// @param element
-///   A BSON element.
-///
-/// @returns A JSON key-value pair.
-///
-BSONCXX_API std::string BSONCXX_CALL to_json(document::element element);
```

Comment:
@xdg @Machyne @hanumantmk  I filed https://jira.mongodb.org/browse/CXX-997 to capture this, which would make the feature request here trivial.

I think we should commit this as is. Any further comments?